CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ elseif(${CMAKE_HOST_SYSTEM_PROCESSOR} MATCHES "parisc.*")
 	set(HARDINFO_ARCH "parisc")
 elseif(${CMAKE_HOST_SYSTEM_PROCESSOR} MATCHES "sparc{32,64}")
 	set(HARDINFO_ARCH "sparc")
+elseif(${CMAKE_HOST_SYSTEM_PROCESSOR} MATCHES "aarch64*")
+	set(HARDINFO_ARCH "aarch64")
```

Comment:
Will it work better here if setting to "arm"? I don't have any aarch64 devices to test this, but there have been a few changes to the arm information gathering a few weeks ago, and it might work fine for aarch64.